pkg/orderedmap/map.go

```diff
@@ -72,6 +72,16 @@ func (m *Map) Iterate(iterFunc func(k, v interface{})) {
 	}
 }
 
+func (m *Map) IterateWithErr(iterFunc func(k, v interface{}) error) (err error) {
```

Comment:
looks like below IterateErr does the right thing already.

Reply:
_literally_ right below.

Fixed.